src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+function slidingTree() {
+  const fruits = document.querySelectorAll('li');
```

Comment:
а если там будут не фрукты? код должен быть понятным независимо от контента страницы
```suggestion
  const listItems = document.querySelectorAll('li');
```